src/index.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
+import App from './App/App';
 import reportWebVitals from './reportWebVitals';
+import './Sass/main.scss';
```

Comment:
Are you planning to put all the sass in one folder?

Reply:
The main.scss imports all the other Scss files so when we only have to call it in index.tsx.

Reply:
Let's talk about this one too. I think there is a better file structure we can use for this